Added quickobs

philclifford opened the pull request:
since I never got to sort my Mullvad VPN fix update for you (thanks for picking it up) at least I can try and save you having to deal with this one from Yannick :smile:

natanjunges commented on 01-main/packages/quickobs:
```diff
@@ -0,0 +1,11 @@
+DEFVER=1
+CODENAMES_SUPPORTED="focal jammy kinetic"
+ARCHS_SUPPORTED="amd64"
```
This is already the default, so you can remove it.

natanjunges commented:
The test is failing because it is trying to install other packages, perhaps because you need to resynchronize your branch. We should try to improve the test so that it does not happen.

natanjunges commented:
https://github.com/natanjunges/deb-get/actions/runs/3269581986/jobs/5377243956#step:5:20
`wavebox` *is* being detected as modified (and so are `lutris`, `vivaldi-stable` and `webex`), and it is not a dependency of `quickobs`.

philclifford commented:
hmm .... PR showed only changes in quickobs ... that branch had synced your commits re webex, vivaldi etc ... but the test decided they were "changed" .  Rebased and pushed : same (but test passed) .  Rebased again (using gitkraken)  (lots of skipping of "already applied patch") - pushed.  Test ran again and only detected quickobs this time (yay!)  
Odd.  

![image](https://user-images.githubusercontent.com/8797027/196516451-c085d0f5-2d46-414b-9f6c-16c6b1b503bb.png)

Anyway,  sorted now I hope - sorry for the hassle and my confusion.

natanjunges commented:
I think there might be a better git magic that would not have this problem, but I need more experimenting.
